packages/charts-components/templates/amd-script-tag/index.html

```diff
@@ -0,0 +1,64 @@
+<!doctype html>
+<html lang="en" dir="ltr">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+    <title>Charts components AMD template</title>
+    <style>
+        .chart-container {
+            height: 800px;
+        }
+    </style>
+
+    <link rel="icon" href="data:;base64,=" />
+
+    <!-- Load the ArcGIS Maps SDK for JavaScript -->
+    <link rel="stylesheet" href="https://js.arcgis.com/4.28/esri/themes/dark/main.css" />
+    <script src="https://js.arcgis.com/4.28/"></script>
+
+    <!-- Load the Charts Components -->
+    <script type="module" src="https://jsdev.arcgis.com/charts-components/4.29/arcgis-charts-components.esm.js"></script>
+</head>
+
+<body>
+    <arcgis-charts-scatter-plot id="scatter-plot" class="chart-container"></arcgis-charts-scatter-plot>
+    <script>
+        /**
+         * Simple data loader against public data.
+         * For the test data, we need a feature layer.
+         */
+        async function loadFeatureLayerItem() {
+            const WebMap = await $arcgis.import("esri/WebMap");
```

Comment:
We have it documented in the components programming patterns page. The coding component templates also uses the global `$arcgis`

Reply:
> We have it documented in the components programming patterns page

True, I almost forgot about that since it's kinda buried in the Using a proxy section. I'll open an issue to revisit its positioning on the page for after release.